phpstan-baseline.neon

```diff
@@ -1,6 +1,12 @@
 parameters:
-    ignoreErrors:
-        - # NEXT_MAJOR: Remove this issue.
-            message: "#^Call to static method getInstance\\(\\) on an unknown class Sonata\\\\EasyExtendsBundle\\\\Mapper\\\\DoctrineCollector\\.$#"
-            count: 1
-            path: src/DependencyInjection/SonataMediaExtension.php
+	ignoreErrors:
+		-
```

Comment:
Please, use spaces instead of tabs.

Reply:
I didn't edit the output from the generated baseline.
I'll perform the requested replace.